package.json

```diff
@@ -75,6 +75,11 @@
                 "category": "ev3dev"
             },
             {
+                "command": "ev3devBrowser.captureScreenshot",
```

Comment:
Just looked at this again and this command should not be showing up in the command palette. Not sure what's up. Perhaps a vscode bug.